src/citrine/_rest/collection.py

```diff
@@ -30,9 +30,9 @@ class Collection(Generic[ResourceType], Pageable):
     _collection_key: str = 'entries'
     _paginator: Paginator = Paginator()
 
-    def _put_module_ref(self, subpath: str, workflow_id: UUID):
+    def _put_module_ref(self, subpath: str, uid: UUID):
```

Comment:
since this method is more generic than just workflows

Reply:
I noticed in other methods you changed the signature to `uid: Union[UUID, str]`, should we change it here too? (this is a nitpick, feel free to ignore)

Reply:
yes